homeassistant/helpers/selector.py

```diff
@@ -153,7 +156,7 @@ class EntityFilterSelectorConfig(TypedDict, total=False):
     integration: str
     domain: str | list[str]
     device_class: str | list[str]
-    supported_features: list[str]
+    supported_features: list[int | str | list[str]]
```

Comment:
This will also break the frontend. We should make sure that integers are converted back to be strings and that we don't change the type.

Reply:
This is not the case, the `_validate_supported_features` validator transforms entity features given by strings to their integer value. The serialized selector sent to frontend has the int, not the strings.